README.md

```diff
@@ -9,15 +9,21 @@ lifecycle management of Clumio resources.
 -	[Go](https://golang.org/doc/install) >= 1.17
 
 ## Installing the provider
+Terraform Provider Clumio is available in the [Terraform Registry](https://registry.terraform.io/providers/clumio-code/clumio/latest) and it will get installed on ```terraform init```.
```

Comment:
Nit: Should it be:

```
The Clumio Terraform Provider is available in the [Terraform Registry](https://registry.terraform.io/providers/clumio-code/clumio/latest).

The following terraform block will install the provider upon "terraform init":
```

I took out "in the configuration file" as I wasn't sure if it needs to be in a configuration file or just in any .tf file (e.g. main.tf even).